adapter-base-quarkus/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-adapter-base-quarkus/reflection-config.json

```diff
@@ -13,5 +13,70 @@
     "name" : "org.eclipse.hono.adapter.resourcelimits.DataVolumeAsyncCacheLoader",
     "allDeclaredMethods" : true,
     "allPublicMethods" : true
+  },
```

Comment:
it should be sufficient to only add the reflection config to the *service-base-quarkus* module as *adapter-base-quarkus* depends on it and thus pulls it in anyway

Reply:
Good point, I've adapted the commit accordingly.